SchedulerBot/SchedulerBot.Business.Entities/ExecutionResult/ExecutionErrorCodes.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace SchedulerBot.Business.Entities
+{
+	/// <summary>
+	/// Execution Error Codes. This enum used inside <see cref="ExecutionResult"/>.
+	/// </summary>
+	public enum ExecutionErrorCodes
+	{
+		None = 0,
```

Comment:
It would be nice to add xml documentation for the enum values as well.

Reply:
done